src/absil/il.fs

```diff
@@ -18,6 +18,7 @@ open System.Collections
  
 let logging = false 
 
+#if FX_RUNNING_ON_MONO
```

Comment:
Running on Mono requires a bunch of APIs that don't exist on coreclr and so won't compile, the variable approach would mean porting a bunch of code to portable that I stand no chance of testing.  Effectively the code in the #if RUNNING_ON_MONO still needs porting which we can do later ... if it's even necessary I expect that mono will do a pretty decent job of running portable code.